content/en/docs/release-notes/release-notes-1.7.md

```diff
@@ -7,12 +7,117 @@ type: "docs"
+specifying additional supplementary formats of issued certificates and their
+private key. There are currently two supported additional output formats:
+`CombinedPEM` and `DER`. Both output formats can be specified on the same
+Certificate.
+Read [Additional Certificate Output Formats] for more details and
+thanks to [@seuf](https://github.com/seuf) for getting this across the line!
+
+[Additional Certificate Output Formats]: ../../usage/certificate/#additional-certificate-output-formats
+
+### Removal of Deprecated APIs
+
+In 1.7 the cert-manager API versions v1alpha2, v1alpha3, and v1beta1 have been removed from the custom resource definitions (CRDs).
+You will notice that the YAML manifest files are much smaller as a result.
+These APIs have been deprecated since 1.5.
+
+In this release we have added a new sub-command to the cert-manager CLI (`cmctl upgrade migrate-api-version`),
+which you SHOULD run BEFORE upgrading cert-manager to 1.7.
+Please read [Removing Deprecated API Resources] for full instructions.
+
+### Server-Side Apply
+
+This is the first version of cert-manager which relies on [Server-Side Apply].
+We are using it to properly manage the annotations and labels on the TLS Secret.
+For this reason cert-manager 1.7 requires at least Kubernetes 1.18.
+
+[Server-Side Apply]: https://kubernetes.io/docs/reference/using-api/server-side-apply/
+
+### Configuration Files
+
+In this release we introduce a new configuration file for the cert-manager-webhook.
+Instead of configuring the webhook using command line flags,
+you can now modify the webhook Deployment to mount a ConfigMap
+containing a configuration file.
+Read the [WebhookConfiguration Schema] for more information.
+
+In future releases we will introduce configuration files for the other cert-manager components: controller-manager and cainjector.
+
+[WebhookConfiguration Schema]: https://cert-manager.io/next-docs/reference/api-docs/#webhook.config.cert-manager.io/v1alpha1.WebhookConfiguration
+
+### Non-bazel Development
+
+We aim to stop using `bazel` for building and testing cert-manager.
+It is too difficult for new contributors and frustrates many of our regular contributors.
+So we have been fixing the cert-manager Makefile and trying to make it possible to run the unit-tests
+with `go test ./cmd/... ./internal/... ./pkg/...` etc.
+
+## Community
+
+Thanks again to all open-source contributors with commits in this release, including:
+
+- [@Adphi](https://github.com/Adphi)
+- [@devholic](https://github.com/devholic)
+- [@johnwchadwick](https://github.com/johnwchadwick)
+- [@jsoref](https://github.com/jsoref)
+- [@jwenz723](https://github.com/jwenz723)
+- [@seuf](https://github.com/seuf)
+- [@thirdeyenick](https://github.com/thirdeyenick)
+
+And thanks as usual to [coderanger](https://github.com/coderanger) for helping people
+out on the Slack `#cert-manager` channel; it's a huge help and much appreciated.
```

Comment:
love it :ok_hand:

Reply:
Should `#cert-manager` be a link to somewhere that helps people get to slack?

Reply:
Done.